image/templates/helm/stackrox-secured-cluster/internal/config-shape.yaml

```diff
@@ -141,6 +142,24 @@ customize:
     podLabels: {} # dict
     podAnnotations: {} # dict
     envVars: {} # dict
+  scanner-v4-indexer:
+    labels: {} # dict
+    annotations: {} # dict
+    podLabels: {} # dict
+    podAnnotations: {} # dict
+    envVars: {} # dict
+  scanner-v4-matcher:
```

Comment:
don't need anything related to matcher

Reply:
Ah, because matcher is not to be installed on secured clusters, get it.